tasks/basic-sanity-check-task.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: basic-sanity-check
+spec:
+  stepTemplate:
+    volumeMounts:
+      - name: project
+        mountPath: /workspace/source/image
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+      default: quay.io/cvpops/test-index:v4.10
+  steps:
+    - name: clone-git-repo
+      image: alpine/git
+      script: "git clone https://github.com/dirgim/hacbs-conftest.git /project"
```

Comment:
Personally I would go with the docker image build from dirgim/hacbs-conftest.git, you would have an image with complete environment (skopeo, conftest and policies) -> easy to do overall test. You are managing content of the image and can easily revert to previous image.

Reply:
Image updated so we have all in one place.